internal/consumer/consumer.go

```diff
@@ -671,11 +672,17 @@ func CreateConsumer(config ConsumerConfig) (*Consumer, error) {
 	started := time.Now()
 	consumer.logger = config.Logger.WithPrefix("[consumer]")
 	consumer.started = &started
-	consumer.max = config.MaxAckPending
 	consumer.ctx = ctx
 	consumer.cancel = cancel
 	consumer.conn = nc
 	consumer.driver = config.Driver
+	consumer.max = config.MaxAckPending
+	if consumer.driver != nil {
+		driverMaxSize := consumer.driver.MaxBatchSize()
```

Comment:
since this is a broad change for all EDS drivers, what is the MaxBatchSize set to for other drivers?